apps/aecore/src/aec_genesis_block_settings.erl

```
@@ -0,0 +1,25 @@
+-module(aec_genesis_block_settings).
+
+-export([dir/0,
+         preset_accounts/0]).
+
+dir() ->
+    case application:get_env(aecore, genesis_settings_dir) of
+        undefined ->
+            filename:join(code:priv_dir(aecore), "genesis");
+      {ok, GenSettingsDir} ->
+            GenSettingsDir
+    end.
+
+preset_accounts() ->
+    PresetAccountsFile = filename:join(dir(), "accounts.json"),
+    case file:read_file(PresetAccountsFile) of
+        {error, _Err} ->
+            [];
```

Do we really want to silently fail here?